Modelica/Electrical/Analog/Lines/M_OLine.mo

```diff
@@ -26,6 +26,8 @@ public
     each final min=Modelica.Constants.small,
     each unit="F/m") = {2.38e-11,1.01e-10,8.56e-11,5.09e-12,2.71e-11,2.09e-11,
     7.16e-11,1.83e-11,1.23e-10,2.07e-11} "Capacitance per meter";
+  parameter Boolean useInternalGround=true "= true if internal ground is used, otherwise use reference pin"
```

Comment:
We have a dilemma here, since our guideline states:

> Introducing a new name in the public section of a class (model, package, ...) or in any section of a partial class is **not** allowed

Reply:
I don't think it is a big issue here. And as you say, it is only a recommendation. The LO's should decide. In the worst case it will be not part of 4.0.1 but 4.1.0 (whenever that is ;-) )

Reply:
Actually the project leader needs to decide if this can simply get merged, or need a mention in the release notes in addition or simply should not get merged.

Reply:
> Actually the project leader needs to decide if this can simply get merged, or need a mention in the release notes in addition or simply should not get merged.

I doubt that @MartinOtter will comment on that. So if we (@dietmarw, @christiankral and @AHaumer) agree on the solution, it should be ok.

Reply:
That might be the pragmatic solution but not the correct one especially since this is regarding a maintenance release. It is unfortunate that we don't have an active project leader but that is not reason enough to bypass this kind of approval.